manager/manager.go

```
@@ -623,3 +625,27 @@ func (self *manager) watchForNewContainers(quit chan error) error {
 	}()
 	return nil
 }
+
+func (self *manager) watchForNewOoms() error {
+	outStream := make(chan *oomparser.OomInstance, 10)
+	oomLog, err := oomparser.New()
+	if err != nil {
+		return err
+	}
+	err = oomLog.StreamOoms(outStream)
+	if err != nil {
+		glog.Errorf("error streaming OOMs: %v", err)
```

here we can return the error